interview-questions.md

```diff
@@ -9,41 +9,41 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
The interview questions are a great opportunity to practice writing what you would like to say to an interviewer. You want to share your personality, how you research and understand information, as well as your experiences.